src/Tags/Concerns/GetsQueryResults.php

```diff
@@ -58,4 +62,15 @@ protected function queryPaginationFriendlyOffset($query, $offset)
 
         $query->whereNotin('id', $offsetIds);
     }
+
+    protected function chunkedResults($query, $chunkSize)
+    {
+        $key = $this->params->get('chunk_key', 'chunk');
```

Comment:
Should this just use `as` instead of `chunk_key`?

Reply:
Yeah I guess it could - makes sense.